### PR TITLE
shared fields api calls in XBlock SDK scenario demo 

### PR DESCRIPTION
**Pull request by beardeer**
@pmitros @cpennington @DongwookYoon

Hi team,

Here is a demo of how shared api calls work. Right now, it is as simple as initializing the shared field (line 30), get the value of shared field from a user (line: 45) and set the value (value: 68).

This demo works with my XBlock changes (https://github.com/edx/XBlock/tree/xiaolu/shared-field)

Please let me know how you feels. Thanks!

### CODE REVIEW

**Comment by pmitros** [comment]
Overall, this looks good. One nit: the names should mirror. shared_user_state is a little confusing. It sounds like user_state_summary. The prefix -- RemoteScope. -- is fine to indicate it is what you're remoting. 

I'd also prefer named parameters. `user="2"` rather than just `"2"`. It makes it clear what is being queried in the code, and is a little bit cleaner when sharing on multiple axes.

**Comment by DongwookYoon** [comment]
This looks good to me.

Dongwook

On Mon, Jul 13, 2015 at 1:52 PM, Piotr Mitros <notifications@github.com>
wrote:

> Overall, this looks good. One nit: the names should mirror.
> shared_user_state is a little confusing. It sounds like user_state_summary.
> The prefix -- RemoteScope. -- is fine to indicate it is what you're
> remoting.
>
> I'd also prefer named parameters. user="2" rather than just "2". It makes
> it clear what is being queried in the code, and is a little bit cleaner
> when sharing on multiple axes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/beardeer/shared_field_demo/pull/1#issuecomment-121004977>
> .
>



-- 
--Dongwook


Dongwook Yoon
PhD Student
Information Science Department
Cornell University
T: +1-408-444-2877
E: dy252@cornell.edu

File: shared_field_demo/shared_field_demo.py [hunk 1]
```diff
@@ -39,7 +40,9 @@ def student_view(self, context=None):
         The primary view of the SharedFieldDemoXBlock, shown to students
         when viewing courses.
         """
-
+        # get the shared field data value
+        # user selector ("2") is hard-coded
+        self.user_id = self.scope_ids.user_id
         self.shared_num_value = self.shared_num.get(self, "2")
```
Comment (cpennington) [inline shared_field_demo/shared_field_demo.py]: I'd prefer if this was written (and perhaps required to be written) as `user="2"`. Without that, the code gets very confusing to read.

Comment (cpennington) [inline shared_field_demo/shared_field_demo.py]: Especially in the case where the `Query` object can query across both users and blocks.

**Pull request closed without merging**